lib/core/test/unit/Cardano/WalletSpec.hs

```diff
@@ -321,6 +338,28 @@ walletKeyIsReencrypted (wid, wname) (xprv, pwd) newPwd =
         [ TxOut (Address "destination") (Coin 14) ]
         []
 
+walletListTransactionsSorted
+    :: (WalletId, WalletName)
+    -> Map (Hash "Tx") (Tx, TxMeta)
+    -> Property
+walletListTransactionsSorted (wid, wname) history =
+    monadicIO $ liftIO $ do
+        let state = Map.insert (Address "source") minBound mempty
+        let wallet = (wid, wname, DummyState state)
```

Comment:
What's the need for an initial non empty state here? I think it could just be `mempty`, couldn't it ? Or even better, just arbitrarily pulled from the arguments `(WalletId, WalletName, DummyState)` ... I believe you just copy/pasted from the test above but the use-case is rather different in the test above ...